src/main/java/liuni/KeyHandler.java

```diff
@@ -94,26 +94,18 @@ public void getTwitterKeys(Element element) {
     }
 
     public void setupTwitter() {
-        try {
-            writer.write("debug=false\n");
-
-            writer.write("oauth.consumerKey=" + conKey + "\n");
-            writer.write("oauth.consumerSecret=" + conSec + "\n");
-            writer.write("oauth.accessToken=" + accToken + "\n");
-            writer.write("oauth.accessTokenSecret=" + accSec + "\n");
-        }
-        catch (IOException e) {
-            System.out.println("Error occurred when setting up twitter4j.properties.");
-            e.printStackTrace();
-        }
-        finally {
+        if (writer != null) {
             try {
-                if (writer != null) {
-                    writer.close();
-                }
-            }
-            catch (IOException e) {
-                System.out.println("Error occurred when closing the Buffered writer.");
+                writer.write("debug=false\n");
+
+                writer.write("oauth.consumerKey=" + conKey + "\n");
+                writer.write("oauth.consumerSecret=" + conSec + "\n");
+                writer.write("oauth.accessToken=" + accToken + "\n");
+                writer.write("oauth.accessTokenSecret=" + accSec + "\n");
```

Comment:
if writer.write throws an exception...you'll never close the writer which is not a good practice.  Let's not go with this solution as a way for code coverage.  The test test the existing code, not modifying code for code coverage